.github/workflows/fetch-people-profiles.yml

```diff
@@ -0,0 +1,74 @@
+name: Weekly - Fetch SSW People Profiles
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * TUE"
+
+env:
+  TEMPORARY_FOLDER: temp-profiles
+  SSW_PEOPLE_BASE_URL: https://ssw.com.au/people
+
+jobs:
+  fetch-and-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v4
+
+      - name: Checkout People Profiles
+        uses: actions/checkout@v4
+        with:
+          repository: "SSWConsulting/SSW.People.Profiles"
+          path: ${{ env.TEMPORARY_FOLDER}}
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
```

Comment:
can you use the version specified in the `.nvmrc`

Reply:
Done - I've updated the action to use `lts/*`

Reply:
Re-done - referencing `.npmrc`